english/swami-vivekananda/teachings-of-swami-vivekananda/quotes.md

```diff
@@ -38,6 +38,38 @@ Infinite faith and strength are the only conditions of success.
 
 
 
+
+ GOLDEN RULES from Swami Vivekananda
+
+1.Who is Helping You, Don’t Forget them.
+  Who is Loving you, Don’t Hate them .
+  Who is Believing you, Don’t Cheat them.
+
+2. Anything that makes you weak physically, intellectually and spiritually, reject as poison. Swami Vivekananda Motivational quote

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.